client/src/pages/private/SiteViewDetails.js

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useState } from 'react';
+import Grid from '@material-ui/core/Grid';
+import { Page, CheckPermissions } from '../../components/generic';
+import { Box, Typography } from '@material-ui/core';
+import { scrollUp } from '../../utils';
+import store from 'store';
+import { Link } from 'react-router-dom';
+import routes from '../../constants/routes';
+
+
+export default (props) => {
```

Comment:
Same here, only `props` seems too broad.

Reply:
I fixed this one up.